Feature/copy across loop permissions

appsol opened the pull request:
### Summary
https://trello.com/c/shIP4udB

- Git cherry-pick commits from https://github.com/the-leeds-repo/api/pull/15/commits
- Ran all tests, updating current app to align with new Role management

### Development checklist
- [ ] Changes have been made to the API?
  - [ ] If so, the OpenAPI specification has been updated
- [ ] Database migrations have been added?
  - [ ] If so, the MySQL Workbench ERD has been updated
- [x] The code has been linted `./develop composer fix:style`

### Release checklist
NA

### Notes
NA

matthew-inamdar reviewed (changes requested):
Looking good 👍 

- There are some failing tests to look at.
- The service import controller must no longer create any roles (org admins will now implicitly be service admins/workers).
- The organisation import controller must no longer create any roles (super/global admins will now implicitly be org admins).

appsol commented:
- Organisation\ImportController does not create admins
- Service\ImportController does not create admins
- Updated tests to match
- Fixed failing tests

matthew-inamdar commented:
LGTM 👍
